include/mbgl/style/expression/type.hpp

```diff
@@ -73,6 +73,11 @@ struct FormattedType {
     bool operator==(const FormattedType&) const { return true; }
 };
 
+struct ImageType {
+    constexpr ImageType(){}; // NOLINT
```

Comment:
why is that needed^?

Reply:
so that it can be used as literal in constexpr context
`constexpr ImageType Image;`

Reply:
changed to default constructor